Core/clim-basic/drawing/design.lisp

```diff
@@ -300,6 +300,10 @@
         (symbol-value dynamic-variable-symbol)
         default-ink)))
 
+(defmethod color-rgba ((ink indirect-ink))
+  (let ((ink (indirect-ink-ink ink)))
+    (color-rgba ink)))
```

Comment:
This is pretty testable, so if you don't mind …

Reply:
fixed